README.md

```diff
@@ -134,7 +134,7 @@ Amoro is built using Maven with Java 1.8 and Java 17(only for `trino` module).
 * To skip tests: `mvn -DskipTests package -P toolchain`
 * To package without trino module and JAVA 17 dependency: `mvn clean package -DskipTests -pl '!trino'`
 * To build with hadoop 2.x(the default is 3.x) `mvn clean package -DskipTests -Dhadoop=v2`
-* To indicate flink version for optimizer(the default is 1.14, 1.15 and 1.16 are available)
+* To indicate flink version for optimizer(the default is 1.15, 1.16 and 1.17 are available)
```

Comment:
The Flink optimizer module supports specifying the dependent Flink version through profiles. Currently, versions 1.14/1.15/1.16 are supported, so there is no need to update the documentation. By the way, we can consider upgrading the version in the profile and then modifying the documentation here.

Reply:
ref:https://github.com/NetEase/amoro/blob/master/ams/optimizer/flink-optimizer/pom.xml#L257

Reply:
We can then keep in sync with the amoro flink connector version